pkg/kubectl/cmd/create.go

```diff
@@ -167,6 +168,23 @@ See http://releases.k8s.io/HEAD/docs/user-guide/services-firewalls.md for more d
 				makePortsString(obj.Spec.Ports, true))
 			out.Write([]byte(msg))
 		}
+
+		_, ok := obj.Annotations[service.AnnotationLoadBalancerSourceRangesKey]
+		if ok && obj.Spec.SourceRange != "" {
+			msg := fmt.Sprintf(`You have specified both sourceRange field and annotation [service.beta.kubernetes.io/load-balancer-source-ranges].
```

Comment:
are we issuing this error in other places?  I like the idea, but I haven't seen it before.